cli/serve/serve.go

```diff
@@ -94,9 +95,23 @@ var v1Endpoints = map[string]func() (http.Handler, error){
 	"scaninfo": func() (http.Handler, error) {
 		return scan.NewInfoHandler(), nil
 	},
+}
 
+var staticEndpoints = map[string]func() (http.Handler, error){
 	"/": func() (http.Handler, error) {
-		return http.FileServer(rice.MustFindBox("static").HTTPBox()), nil
+		//the default rice order doesn't include working directory
+		order := []rice.LocateMethod{
+			rice.LocateEmbedded,
+			rice.LocateAppended,
+			rice.LocateFS,
+			rice.LocateWorkingDirectory,
+		}
+		c := &rice.Config{LocateOrder: order}
+		box, err := c.FindBox(staticDir)
+		if err != nil {
+			panic(err)
```

Comment:
I agree I just didn't want to change the existing behavior. I will make it not panic.